index.rst

```diff
@@ -96,7 +96,9 @@ The main documentation for the site is organized into the following sections:
    tutorials/vr/index
    tutorials/plugins/index
    tutorials/platform/index
+   tutorials/best_practices/index
    tutorials/performance/index
+   tutorials/export/index
```

Comment:
The Exporting section might be worth having further up, as exporting a game is not something that one should do *last*, it's actually something that should come up early on during development.

There's actually an "Exporting" tutorial right after Dodge the Creeps, since users will typically want to export their first game to see their finished playable project.

That being said I'm not sure *where* exactly this section should go, and it doesn't impact the actual moving of files so this shouldn't prevent merging.

Reply:
We should probably review and swap all sections. I'd just need to know if it should be by the order in which we'd recommend to read, in order of how likely people are to read it, etc. Alphabetical could also be an option.